UnitsNet/UnitsNet.Net35.csproj

```diff
@@ -40,7 +40,158 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs;" Exclude="obj\**;**\*.WindowsRuntimeComponent.cs" />
+    <Compile Include="AmbiguousUnitParseException.cs" />
```

Comment:
For some reason, even after restarting visual studio, the files were not yet present
I've reverted the changes (manually, not sure how to do this with git)